lib/osf-components/addon/components/node-card/template.hbs

```diff
@@ -86,7 +105,7 @@
             {{/if}}
         {{else}}
             {{#content-placeholders as |placeholder|}}
-                {{placeholder.text lines=2}}
+                {{placeholder.text lines=3}}
```

Comment:
This lines up well for registrations, but on the forks page 2 lines would be closer to the height of the rendered card:
![kapture 2018-10-04 at 13 37 31](https://user-images.githubusercontent.com/6776190/46492199-daa75f00-c7da-11e8-8aff-424c51efa7ae.gif)
Might not be worth pursuing now, but ideally all these cards would be the same height.

Reply:
Ohh, I see. I think would be better to leave a blank space where the description would be, but that's definitely out of scope of this PR 👍